components/textbook-beta/SummerSchoolCoursePages.vue

```diff
@@ -0,0 +1,340 @@
+<template>
+  <section class="summer-school-course-pages-section">
+    <div>
+      <h2>
+        Course pages
+      </h2>
+      <nav class="summer-school-course-pages-section__nav">
+        <ul class="courses">
+          <li
+            v-for="{ label, segment, url } in courses"
+            :key="label"
+            class="course"
+          >
+            <BasicLink
+              class="course__entry"
+              :class="{ active: label === activeCourseLabel }"
+              :segment="segment"
+              :url="url"
+              @mouseenter="selectCourse(label)"
+            >
+              {{ label }}
+            </BasicLink>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <main class="summer-school-course-pages-section__main">
+      <div class="summer-school-course-pages-section__main__title-wrapper">
+        <div class="summer-school-course-pages-section__main__title copy__subtitle">
+          Page preview
+        </div>
+      </div>
+      <div class="summer-school-course-pages-section__main__preview">
+        <BasicLink :url="activeCourse.url">
+          <img
+            class="summer-school-course-pages-section__main__preview__image"
+            :src="activeCoursePreviewImage"
+          >
+        </BasicLink>
+        <AppCta
+          v-bind="activeCourse"
+          label="Go to page"
+          class="summer-school-course-pages-section__main__preview__cta"
+        />
+      </div>
+    </main>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { CtaClickedEventProp } from '~/constants/segment'
+
+interface Course {
+  image: string;
+  label: string;
+  segment: CtaClickedEventProp;
+  url: string;
+}
+
+@Component
+export default class SummerSchoolCoursePagesSection extends Vue {
+  activeCourseLabel = '';
+
+  courses: Course[] = [
```

Comment:
Similar to above, since this was a small task, the refactoring was out of scope... I created issues here:

https://github.com/Qiskit/qiskit.org/issues/2168
https://github.com/Qiskit/qiskit.org/issues/2167